packages/components/src/progress-bar/stories/index.story.tsx

```diff
@@ -30,3 +31,9 @@ const Template: StoryFn< typeof ProgressBar > = ( { ...args } ) => {
 
 export const Default: StoryFn< typeof ProgressBar > = Template.bind( {} );
 Default.args = {};
+
+// The ProgressBar width can be overriden (in pixels) and it will expand to that size if
+// the parent has enough horizontal space. It is interpolated into the `width` property
```

Comment:
I guess you mean `prop`?

```suggestion
// the parent has enough horizontal space. It is interpolated into the `width` prop
```

Reply:
It's the same but unabbreviated, but I get your point on leveraging React's vocab 👍🏻